_includes/benefits.html

```diff
@@ -6,38 +6,32 @@ <h2 class="section-heading text-center">What We Offer</h2>
                 <h3 class="section-subheading text-muted text-center">We believe in building "better" men</h3>
             </div>
             <div class="col-md-6">
-                <div class="benefits-text-header">
-                    <div class="benefits-text-header-text">
-                        🤗 Support and Connection
-                    </div>
-                </div>
-                <div>
+                <div class="benefit">
+                    <h3>
+                        <i>🤗</i> Support and Connection
+                    </h3>
                     <ul>
-                        <li>You are <b>heard and respected</b>. Everyone listens to your words. What you say is confidential</li>
-                        <li>You are <b>valued</b> and <b>respected</b>by a circle of men who help each other in realizing their dreams</li>
+                        <li>You are <b>heard and respected</b>. Everyone listens to your words. What you say is confidential.</li>
+                        <li>You are <b>valued</b> and <b>respected</b>by a circle of men who help each other in realizing their dreams.</li>
                     </ul>
                 </div>
-                <div class="benefits-text-header">
-                    <div class="benefits-text-header-text">
-                        🌱 Growth and Belonging
-                    </div>
-                </div>
-                <div>
+                <div class="benefit">
+                    <h3>
+                        <i>🌱</i> Growth and Belonging
```

Comment:
Do you need the italic emoji for the effect?

Reply:
I followed the same pattern as FontAwesome (in some version), where `<I>` is just a generic html tag, not to mean "italic". I de-italicized it in css.

Reply:
It could just as well be <span>, but <I> is shorter. Lazy dev.